native/CMakeLists.txt

```diff
@@ -13,6 +13,7 @@ set(ENGINE_NAME cocos_engine)
 cc_set_if_undefined(USE_SE_V8                ON)
 cc_set_if_undefined(USE_SE_SM                OFF)
 cc_set_if_undefined(USE_V8_DEBUGGER          ON)
+cc_set_if_undefined(USE_SE_DEBUGGER_FORCE    OFF)
```

Comment:
Could the name be  USE_V8_DEBUGGER_FORCE ?

Reply:
`USE_SE_DEBUGGER_FORCE` is for all possible script engines. And yes USE_V8_DEBUGGER_FORCE more accurate for now